client/app/pages/users/UsersList.jsx

```diff
@@ -116,6 +120,45 @@ class UsersList extends React.Component {
     }),
   ];
 
+  componentDidMount() {
+    if ($location.path() === '/users/new') {
+      this.showCreateUserDialog();
+    }
+  }
+
+  createUser = values => new Promise((resolve, reject) => {
+    User.create(values, (user) => {
```

Comment:
```javascript
createUser = values => User.create(values).$promise
  .then((user) => {
    // code from success callback
  })
  .catch((error) => {
    if (!(error instanceof Error)) {
      error = new Error(get(error, 'data.message', 'Failed saving.'));
    }
    throw error; // Promise should be rejected with Error instance
  });
```

Reply:
I'm not sure if I did that wrong (I think not), but the ErrorHandler seems to log the thrown error in any case (Even though I catch the error :man_shrugging: and the rest working properly). I'm using `return Promise.reject(error)` instead of `throw error` to avoid this.